src/components/YourFeed.tsx

```diff
@@ -42,15 +44,19 @@ export interface DispatchProps {
     onDeletePost: (post: Post) => void;
     onSavePost: (post: Post) => void;
     onSharePost: (post: Post) => void;
-    onUnfollowFeed: (feedUrl: string) => void;
+    onUnfollowFeed: (feed: Feed) => void;
+    onAddFeed: (feed: Feed) => void;
+    onToggleFavorite: (feedUrl: string) => void;
 }
 
 export interface StateProps {
     navigation: any;
     posts: Post[];
+    feeds: Feed[];
+    knownFeeds: Feed[];
     settings: Settings;
-    displayFeedHeader: boolean;
-    showUnfollow: boolean;
+    yourFeedVariant: YourFeedVariant;
+    notOwnFeed: boolean;
```

Comment:
Why is the naming inverse? Usually it's easier to follow the logic if the variable name is direct, e.g. `isOwnFeed`